### PR TITLE
Added website views widget 

### PR DESCRIPTION
**Pull request by rohitinu6**
solves #266 
As mentioned in the issue #266 , I have added the website visitors widget that shows the count of website visitors.

## Screenshots of work:

**Before:**
image

**After:**
image

@dohinafs , please assign the respective issue to me and add the gssoc and hacktoberfest labels

### CODE REVIEW

**Review by Altair-05 — CHANGES_REQUESTED** [review]
The view count is static and does not change with actual visits

**Comment by rohitinu6** [comment]
@Altair-05 , thank you for your review, but it gets updated automatically everytime, someone lands on the page.
You can also check the details over here : https://hits.sh/arcticdelights.netlify.app/

File: index.html [hunk 1]
```diff
@@ -555,7 +555,9 @@ <h4>- Stacy Stone</h4>
 <!-- Footer Section Starts Here -->
 
     <section class="footer" id="footer">
-
+<div>
+    <img src="https://hits.sh/arcticdelights.netlify.app.svg?label=Website%20Visitors&extraCount=6520&color=FE6648&labelColor=white" alt="web-views">
```
Comment (Altair-05) [inline index.html:559]: it is always showing  visitors count  6520

**Review by Altair-05 — APPROVED** [review]

**Comment by dohinafs** [comment]
Kindly issue another pull request or update your version as it is not up to date.

**Comment by rohitinu6** [comment]
@dohinafs , done, pls check

**Review by dohinafs — APPROVED** [review]

**Pull request merged**